app/src/test/java/org/simple/clinic/bloodsugar/history/BloodSugarHistoryScreenUiRendererTest.kt

```diff
@@ -19,4 +22,19 @@ class BloodSugarHistoryScreenUiRendererTest {
     // then
     verifyZeroInteractions(ui)
   }
+
+  @Test
+  fun `when patient is loaded, then show it on the ui`() {
+    // given
+    val patient = PatientMocker.patient(
+        uuid = patientUuid
```

Comment:
Does this need to be on its own line? I generally prefer to put them on the same line if there's only one parameter (Unless that parameter itself is another function or a constructor invocation).